packages/material/src/muicontrols/MuiCheckbox.tsx

```diff
@@ -1,7 +1,7 @@
 /*
   The MIT License
 
-  Copyright (c) 2017-2019 EclipseSource Munich
+  Copyright (c) 2018 EclipseSource Munich
```

Comment:
Outdated year, it needs to be updated for other files as well. Probably it'd be best if you could execute the add-headers scripts to update these occurrences.